Directory.Build.props

```
@@ -42,7 +42,7 @@
     <ContinuousIntegrationBuild Condition="'$(GITHUB_RUN_ID)' != ''">true</ContinuousIntegrationBuild>
     <PackageOutputPath>$(BaseArtifactsPath)pkg/$(Configuration)/</PackageOutputPath>
     <Product>TerraFX.Interop.Windows</Product>
-    <VersionPrefix>10.0.19041</VersionPrefix>
+    <VersionPrefix>10.0.20348</VersionPrefix>
     <VersionSuffix>beta2</VersionSuffix>
```

New version so the beta version should be reset:
```suggestion
    <VersionSuffix>beta1</VersionSuffix>
```